### PR TITLE
ROX-25559: Remove central pvc from roxctl

### PR DESCRIPTION
**Pull request by c-du**
### Description

Remove central pvc from roxctl and rendering

I will add deprecation notice in the change log later.

### User-facing documentation

- [ ] CHANGELOG is updated **OR** update is not needed
- [ ] [documentation PR](https://spaces.redhat.com/display/StackRox/Submitting+a+User+Documentation+Pull+Request) is created and is linked above **OR** is not needed

### Testing

- [ ] inspected CI results

#### Automated testing

- [ ] added unit tests
- [ ] added e2e tests
- [ ] added regression tests
- [ ] added compatibility tests
- [ ] modified existing tests

#### How I validated my change

change me!

### CODE REVIEW

File: roxctl/central/generate/volumes.go [hunk 1]
```diff
@@ -58,9 +53,8 @@ func noVolume(cliEnvironment environment.Environment) *cobra.Command {
 
 func hostPathVolume(cliEnvironment environment.Environment) *cobra.Command {
```
Comment (0x656b694d) [inline roxctl/central/generate/volumes.go:54]: Do we still support `hostPathVolume`?

Comment (c-du) [reply roxctl/central/generate/volumes.go:54]: Yes, it still handles the db hostpath. We are going to deprecate it eventually but not with this PR.

**Review by dashrews78 — APPROVED** [review]

**Pull request merged**